packages/stores/types.ts

```diff
@@ -100,7 +106,9 @@ type TClientStore = {
         };
     };
     account_list: TAccountsList;
-    account_status: GetAccountStatus;
+    account_status: GetAccountStatus & {
+        p2p_status?: string;
+    };
```

Comment:
```suggestion
    account_status: GetAccountStatus;
```
@niloofar-deriv `p2p_status` already exists in `GetAccountStatus` with a better type 🤔 

<img width="444" alt="image" src="https://user-images.githubusercontent.com/72082844/232398547-a3fe8928-b190-4c0d-be6c-8a9fefb5e16d.png">